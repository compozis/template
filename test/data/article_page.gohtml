{{extends "layouts/master.gohtml"}}
{{define "Content"}}ARTICLE{{end}}