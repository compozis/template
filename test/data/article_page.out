<html>
<head>
    <title></title>
</head>
<body>
<div class="content">ARTICLE</div>
<div class="footer">[DEFAULT] Footer</div>
</body>
</html>
